templates/web/base/waste/bulky/items.html

```diff
@@ -9,7 +9,7 @@
   [% INCLUDE 'waste/_address_display.html' %]
 [% END %]
 
-<form class="waste" method="post">
+<form class="waste" method="post" enctype="multipart/form-data">
```

Comment:
I tried adding this in the form class (as in the example in https://metacpan.org/release/GSHANK/HTML-FormHandler-0.40068/view/lib/HTML/FormHandler/Field/Upload.pm), but it didn't seem to do anything, so if this is the solution I will be simultaneously pleased and annoyed it was this simple all along.

Reply:
I can't see how they render that as HTML in the example, but note in our HTML the `<form>` element is baked into the template and is independent of the `FormHandler` instance so that attribute won't have any effect.